<ui version="4.0" >
 <class>KPlato::PertResult</class>
 <widget class="QWidget" name="KPlato::PertResult" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>285</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>Project Schedule:</string>
       </property>
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="indent" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scheduleName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text" >
        <string>Total Float:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalFloat" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DoubleTreeViewBase" name="treeWidgetTaskResult" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleTreeViewBase</class>
   <extends>QFrame</extends>
   <header>kptitemmodelbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
